### ISSUE TITLE
Symfony 2.2 upgrade

### ISSUE THREAD

**Issue by clubdesarrolladores** [xi-project/xi-bundle-breadcrumbs#12]
There is possible upgrade to symfony 2.2?

Thanks.

**Comment by peterhil** [comment]
Hi! I see you made a fork. I doubt I have time or interest right now to update this, as I don't use Symfony2 framework at the moment in any project. But please send a pull request with unit tests and I'll see it through and merge after review.

**Comment by clubdesarrolladores** [comment]
This bundle is awesome, and with a few features more, it can be the best breadcrumb bundle for symfony.

In my fork i change composer to the correct symfony2.2 versions BUT!... i have problems with besimplel18nroutingbundle (seems outdate too).

The phpunit tests fails.

**Comment by puppe0** [comment]
Yes, the fact that there is no Symfony 2.2 compatible version of https://github.com/BeSimple/BeSimpleI18nRoutingBundle makes updating to 2.2 a bit harder. :)

I'll see what I can do to fix this..

**Comment by charlyox** [comment]
Hi there! I'm really interested in this bundle. But my project which is built with symfony 2.3 doesn't seem to accept it, via composer install. I guess this is due to this BeSimple dependency issue . Has somebody around here found a way to fix this?
